caffeinated/admin/extend/events/Extend_Events_Admin_Page.core.php

```diff
@@ -146,17 +107,60 @@ protected function _extend_page_config()
                 'capability' => 'manage_options',
                 'noheader'   => true,
             ),
-        );
-        $this->_page_routes = array_merge($this->_page_routes, $new_page_routes);
-        // partial route/config override
-        $this->_page_config['import_events']['metaboxes'] = $this->_default_espresso_metaboxes;
-        $this->_page_config['create_new']['metaboxes'][] = '_premium_event_editor_meta_boxes';
-        // don't load qTips if using the advanced editor
+        );        // don't load these meta boxes if using the advanced editor
         if (! $this->admin_config->useAdvancedEditor()) {
+            $this->_page_config['create_new']['metaboxes'][] = '_premium_event_editor_meta_boxes';
+            $this->_page_config['edit']['metaboxes'][] = '_premium_event_editor_meta_boxes';
             $this->_page_config['create_new']['qtips'][] = 'EE_Event_Editor_Tips';
             $this->_page_config['edit']['qtips'][] = 'EE_Event_Editor_Tips';
```

Comment:
I hope the JS and CSS assets related to these metaboxes are not loaded.

Reply:
thnx... double checked and found a few that were still being loaded 👍